fedora_bot.py

```diff
@@ -253,11 +297,15 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("-u", "--user", help="Set the username of the Fedora account")
     parser.add_argument("-p", "--password", help="Set the Fedora account password")
+    parser.add_argument("-a", "--apikey", help="Set the Fedora account API key")
     args = parser.parse_args()
 
     fedoras = get_fedora_releases()
 
     for component in components:
+        msg_info(f"Checking for open pull requests of {component}...")
+        get_open_pull_requests(args,component)
```

Comment:
Shall we name this differently? I prefer `get*` functions not to have side effects. Also missing space after `,`.

Reply:
Makes sense. Originally I wanted to create a function `merge_open_pull_requests` and then it turned out to be so little code that I stuck with the get function. I'll rename it.
And the space is missing all over the code, I'll adjust it in a follow-up commit to keep this reviewable.